instrumentation/mysql/src/main/java/brave/mysql/TracingStatementInterceptor.java

```diff
@@ -79,7 +79,7 @@ static void parseServerAddress(Connection connection, Span span) {
         }
       }
       Endpoint.Builder builder = Endpoint.newBuilder().serviceName(remoteServiceName).port(port);
-      if (!builder.parseIp(connection.getHost())) return;
```

Comment:
I think this meant to be "return builder.build()" as the port isn't useful without the IP

Reply:
I will explain. We have mysql connector configured with URL containing DNS name. With current behaviour we see traces like below:
```
api-server    : | GET /api/resource                         |
    api-server:     | SELECT * FROM table    |
```
Second "api-server" entry is related to mysql. It will be convenient to see mysql service name instead of "api-server" there. But because connector is configured without explicit IP new endpoint is not added to current Span.

Reply:
ya I mean the old code just did "return" I think you can get this behavior by only changing it to "return builder.build"

thanks for the clarification though!

Reply:
Do you suggest to return `Endpoint` from `parseServerAddress` method? Currently it is `void`.

Reply:
ah I didn't notice that. your code is simplest